src/models/dft/functional_components.js

```diff
@@ -0,0 +1,388 @@
+/**
+ *  Enum for functional component types
+ */
+export const FC_TYPES = {
+    exchange: "exchange",
+    correlation: "correlation",
+    nonLocalCorrelation: "non-local correlation",
+};
+
+/**
+ * List of functional components
+ * @todo refactor to object of objects with slugs as keys.
+ */
+export const FUNCTIONAL_COMPONENTS = {
```

Comment:
This is a copy of what we have in ESSE. We should have one source of truth.

Reply:
I agree we should have one source of truth. But perhaps definitions like this are better located in MoDe (I think this should be further sketched out in a task). Alternatively, I could also make this object available through the ESSE JS package, sth along the lines of
```javascript
import { FUNCTIONAL_COMPONENTS } from "@exabyte-io/esse/models"
```